standard-covoiturage_openapi.yaml

```diff
@@ -653,14 +832,50 @@ components:
           required:
             - driver
             - price
+            - webUrl
           properties:
-            driver:
-              $ref: '#/components/schemas/User'
+            driverId:
```

Comment:
This change is not compatible with the bookings events flow (https://github.com/fabmob/standard-covoiturage/pull/5) as the passenger / driver will not be known on MaaS's side.